packages/components/htz-components/src/components/NavigationMenu/NavigationMenu.js

```diff
@@ -120,7 +117,17 @@ class NavigationMenu extends React.Component {
     }).isRequired,
   };
 
-  state = { isHovered: false, };
+  state = { isHovered: false, isUser: null, };
+
+  componentDidUpdate() {
+    if (window && typeof window !== 'undefined') {
```

Comment:
If anything, this check should be reversed, but you don't need both.
```suggestion
    if (typeof window !== 'undefined') {
```

But do we even need it at all? `componentDidUpdate` isn't run on the server, so `window` is guaranteed to exist.